app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -195,6 +207,20 @@ def parse_deployment(server, deployment)
         parse_base_item(deployment).merge(specific)
       end
 
+      def parse_messaging(server, messaging, config)
+        data = {
+          :name              => messaging.name,
+          :middleware_server => server,
```

Comment:
it is better to leverage here the `parse_base_item `method, like done here:
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb#L195